index.html

```diff
@@ -46,16 +47,164 @@ <h1>I’m Anthony <br> Glad to see you!</h1>
         </li>
         <li>
           <a href="#">
-            <img src="mobile-images/github.svg" alt="Github icon">
+            <img src="mobile-images/github.svg" alt="Github">
           </a>
         </li>
         <li>
           <a href="#">
-            <img src="mobile-images/peace.png" alt="Angelist icon">
+            <img src="mobile-images/peace.png" alt="Angelist">
           </a>
         </li>
       </ul>
     </section>
+   <!---Projects Section-->
+    <section class="works-section">     
+        <div class="card">
+          <img src="mobile-images/works/SnapshootPortfolio.png" alt="" class="img-center">
+           <h2 class="project-title">Tonic</h2>
+           <div class="card-sub-head">
+            <span class="canopy">CANOPY</span>
+            <i class="fa fa-fw fa-circle"></i>
+            <span class="sub-text"> Back End Dev</span>
+            <i class="fa fa-fw fa-circle"></i>
+            <span class="sub-text"> 2015</span>
```

Comment:
Please use the same font which is the `Poppins` works section. Some of the buttons and headings use `Times New Roman`. Please make sure to follow the right font as per Figma's design.
![font](https://user-images.githubusercontent.com/35504149/159133892-0cd18908-1c0c-426c-a3d0-03d44c2142ef.png)

Reply:
Implemented 👍